src/cromshell/status/command.py

```diff
@@ -0,0 +1,123 @@
+import logging
+import click
+
+import requests
+import tempfile
+import json
+import re
+import fileinput
+import jq
+
+from cromshell.utilities import HTTPUtils
+from cromshell.utilities import IOUtils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="status")
+@click.option(
+    "-q",
+    "--quiet",
+    is_flag=True,
+    help="Suppress Turtle Logo",
+)
+@click.argument('workflow_id')
+@click.pass_obj
+def main(cromwell_config, workflow_id, quiet):
+    """Check the status of a Cromwell job UUID"""
+    click.echo("status")
+    LOGGER.info("status")
+
+    retVal = 0
+    curl_out = "str"
+    tempf = tempfile.TemporaryFile()
+
+    print(cromwell_config.server)
+    # Check if Cromwell Server Backend works
+    HTTPUtils.assertCanCommunicateWithServer(cromwell_config.server)
+
+    # Curl workflow status
+    try:
```

Comment:
I think this try is unnecessary, because you just checked above.
Of course, there's the possibility that in-between these two lines the internet connection or the server died, but that's only in theory I guess @jonn-smith ?